### PR TITLE
Add feature 36 

### PR DESCRIPTION
**Pull request by Awambeng**
Defining threshold values and alert on each metric (CPU, RAM, DISK)

### CODE REVIEW

File: app.py
Comment (stephane-segning) [inline app.py:1]: there's a problem with your implementation. The threshold shouldn't be here, but in the service, when getting them

File: requirements.txt
Comment (stephane-segning) [inline requirements.txt:1]: You shouldn't add all dependencies. Just required one

Comment (stephane-segning) [reply requirements.txt:1]: I mean by that, only top dependencies. The other are transitive

**Review by stephane-segning — CHANGES_REQUESTED** [review]

File: app.py
Comment (Awambeng) [reply app.py:1]: Sir please meaning I should set the threshold in each metric file under the service folder?

File: requirements.txt
Comment (Awambeng) [reply requirements.txt:1]: I added just the psutil dependency in the requirements.txt file

File: app.py
Comment (Awambeng) [reply app.py:1]: Prof does it mean apart from the changes on the threshold my code or implementation is correct?

**Comment by Awambeng** [comment]
Hello sir, I have made the corrections you recommended please will be
waiting for your feedbacks

File: app.py
Comment (stephane-segning) [reply app.py:1]: Yes that's the meaning. Your implementation is actually correct, but at the wrong place

File: app.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from flask import Flask, render_template, jsonify
 from flask_apscheduler import APScheduler
+import psutil
```
Comment (stephane-segning) [inline app.py:3]: You should then remove the import of a thing which is not used, isn't it?

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Good implementation! Good job boy!

**Comment by stephane-segning** [comment]
I think you're going to wait for the following tickets to finish first:
- #30 
- #12

File: app.py (hunk 1 above)
Comment (Awambeng) [reply app.py:3]: yes sir it is

Comment (Awambeng) [reply app.py:3]: I am done updating the imports in the app.py

**Comment by Awambeng** [comment]
> I think you're going to wait for the following tickets to finish first:
> 
> * [Implement tests to validate network monitoring #30](https://github.com/ADORSYS-GIS/monitor-mind/issues/30)
> * [Code a script to monitor disk usage and availability #12](https://github.com/ADORSYS-GIS/monitor-mind/issues/12)

Sir please i don't understand what you want me to do here i just checked on the above tickets and people have already been assigned for that so i don't understand what you want me to do. please if you can throw more explanations it will be good

**Comment by stephane-segning** [comment]
> > I think you're going to wait for the following tickets to finish first:
> > 
> > * [Implement tests to validate network monitoring #30](https://github.com/ADORSYS-GIS/monitor-mind/issues/30)
> > * [Code a script to monitor disk usage and availability #12](https://github.com/ADORSYS-GIS/monitor-mind/issues/12)
> 
> Sir please i don't understand what you want me to do here i just checked on the above tickets and people have already been assigned for that so i don't understand what you want me to do. please if you can throw more explanations it will be good

I was talking about you helping them finish their tickets first before you finish yours. But your ticket is ready to be merged, so proceed first

**Review by stephane-segning — APPROVED** [review]
Good

**Pull request merged**